homeassistant/components/sensor/geizhals.py

```diff
@@ -0,0 +1,142 @@
+"""
+Parse prices of a device from geizhals.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.geizhals/
+"""
+import logging
+from datetime import timedelta
+
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+import homeassistant.helpers.config_validation as cv
+from homeassistant.util import Throttle
+from homeassistant.helpers.entity import Entity
+from homeassistant.const import (CONF_DOMAIN, CONF_FRIENDLY_NAME, CONF_NAME)
+
+REQUIREMENTS = ['beautifulsoup4==4.6.0']
+_LOGGER = logging.getLogger(__name__)
+
+CONF_product_id = 'product_id'
+CONF_regex = 'regex'
+
+ICON = 'mdi:coin'
+MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=120)
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_NAME): cv.string,
+    vol.Required(CONF_FRIENDLY_NAME, default='Price'): cv.string,
```

Comment:
I think it would be better to replace `friendly_name` with something else (perhaps `description`) to avoid confusion with `friendly_name` from the [customize:](https://home-assistant.io/docs/configuration/customizing-devices/).

Also, if the configuration variable is required then no default is needed. Looks like that it could be `vol.Optional`.